x-pack/test/functional/config.js

```diff
@@ -196,6 +197,10 @@ export default async function({ readConfigFile }) {
         pathname: '/app/kibana/',
         hash: '/management/elasticsearch/transform',
       },
+      reporting: {
```

Comment:
Ok, so this is how `navigateToApp('reporting')` works. But I think we shouldn't add it here because Reporting isn't an app.

Reply:
ok, reverted.